src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrInitializer.cs

```diff
@@ -113,21 +113,17 @@ public class VmrInitializer : VmrManagerBase, IVmrInitializer
         _logger.LogInformation("Recursive initialization for {repo} / {sha} finished", mapping.Name, newSha);
     }
 
-    private async Task InitializeRepository(
-        SourceMapping mapping,
-        string? targetRevision,
-        string? targetVersion,
-        CancellationToken cancellationToken)
+    private async Task InitializeRepository(VmrDependencyUpdate update, CancellationToken cancellationToken)
     {
-        _logger.LogInformation("Initializing {name} at {revision}..", mapping.Name, targetRevision ?? mapping.DefaultRef);
+        _logger.LogInformation("Initializing {name} at {revision}..", update.Mapping.Name, update.TargetRevision);
```

Comment:
I think, it's possible to have:
```C#
_logger.LogInformation($"Initializing '{update.Mapping.Name}' at '{update.TargetRevision}'..");
``

Anyway,  please put `name` and `revision` inside `` quotes.

Reply:
the same for Line: 161

Reply:
Ideally, you want to keep the message as a static string with placeholders and keep these as separate parameters (Visual Studio even tells you this) for few reasons, main ones being:
- No interpolation happens unnecessarily when logging doesn't happen (e.g. log level < information)
- The logger could do something with those values (e.g. store them in a separate column so they are searchable)

I used to have these quoted but for this specific statement, I keep copying the SHA a lot from the log and it was better without the quotes actually, so while I usually quote values in traces, here it's not too user friendly.